.github/actions/setup-sql-server/setup-sql-server.ps1

```diff
@@ -0,0 +1,10 @@
+﻿if ($Env:RUNNER_OS -eq "Windows")
```

Comment:
It surely is, hence the description "Sets up SQL Server with Lombiq-recommended defaults, and if you have a different opinion then too bad because we don't care." Or something. This is, for now, mostly for us. If we want to make them usable for a wider audience then we need to [publish them in the marketplace](https://lombiq.atlassian.net/wiki/spaces/DEV/pages/2545614855/Writing+GitHub+Actions+workflows#Ideas-for-the-future) anyway. There are other SQL Server setup actions BTW with differently opinionated approaches, it doesn't necessarily make sense to publish another one.

I kept the hard-coded password for the same reason.